### ISSUE TITLE
How to use Gradient Accumulation with RectifiedAdam

### ISSUE THREAD

**Issue by astariul** [tensorflow/addons#1912]
I'm trying to use Gradient Accumulation with RectifiedAdam optimizer.

How can I modify the code to use gradient accumulation ?

I saw [this](https://github.com/bojone/bert4keras/blob/master/bert4keras/optimizers.py#L647-L697), but it's based on Keras backend so I can't apply it here.

There is also various other implementation for Keras out there, but nothing for Optimizer v2. 

**Any tip on how to implement this feature my self ?**

I'm using Keras fit method.

**Comment by bhack** [comment]
Have you tried with https://github.com/run-ai/runai/?

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
